script/1_0_2current_en_script/Ciel/Day 4/04_01_CIEL04_1C.txt

```diff
@@ -70,7 +70,7 @@ It doesn't change the fact that I killed you with my own hands. That's why――
 <Page17>
 There's no way I can be forgiven.
 It's not acceptable to say "It doesn't matter because she's alive again".
-I, Tohno Shiki, have killed this girl in front of me.
+I, Tohno Shiki, have killed this woman in front of me.
```

Comment:
Whatever we go with above, this is a repeat of the lines before. Same comments apply